extension/chrome/elements/compose.ts

```diff
@@ -84,7 +84,7 @@ Catch.try(async () => {
     });
     to = reply.to;
     from = reply.from;
-    subject = Google.gmail.findHeader(gmailMsg, 'subject') || '';
+    subject = subject || Google.gmail.findHeader(gmailMsg, 'subject') || '';
```

Comment:
I don't know If I fixed it correctly.
What was the reason to re-set subject?

Reply:
There was a subject starting with "Re: " but then you change it to the same one but without "Re: "

Reply:
I'm not sure, I'll have to take a look.

Reply:
I'll try to fix it and push the result. From that you'll see the intention of what is supposed to happen, and if the tests still fail, fix up my fix to work properly.

Reply:
:smiley: